community/iniparser/depends

```diff
@@ -0,0 +1 @@
+musl
```

Comment:
This is an assumed dep; you can just remove the `depends` file

Reply:
Solved